### PR TITLE
Initial steps to connect to prod database

### PR DESCRIPTION
**Pull request by daxmc99**
Add steps for connection to production Postgres database

### CODE REVIEW

**Review by efritz — APPROVED** [review]

File: handbook/engineering/deployments.md [hunk 1]
```diff
@@ -86,6 +86,18 @@ git push origin release
...
 1. Once you have fixed the issue in the `main` branch of [sourcegraph/sourcegraph](https://github.com/sourcegraph/sourcegraph), re-enable auto-deploys by reverting your change to [renovate.json](https://github.com/sourcegraph/deploy-sourcegraph-dot-com/blob/release/renovate.json) from step 1.
 
+### Accessing sourcegraph.com database
+
+Sourcegraph.com utilizes an external HA database. You will need to connect to it directly. The easiest way to do this is through the `gcloud` cli.
+
+To connect to the production database:
+
+```
+  gcloud beta sql connect sg-cloud-732a936743 --user=sg -d sg --project sourcegraph-dev
```
Comment (efritz) [inline handbook/engineering/deployments.md:96]: This requires cloud proxy locally?

Comment (daxmc99) [reply handbook/engineering/deployments.md:96]: It technically spins up the cloud proxy locally and tunnels through it but it is all encapsulated in the command.

Comment (efritz) [reply handbook/engineering/deployments.md:96]: I didn't have the cloud sql binary so I had to install it. There were links in the instructions though so I think it's pretty straightforward.

Comment (daxmc99) [reply handbook/engineering/deployments.md:96]: Ahh ok, good to know

**Review by tsenart — APPROVED** [review]
👌

**Pull request merged**